TaskHouseUnitTests/SkillsControllerUnitTests.cs

```diff
@@ -0,0 +1,216 @@
+using System;
+using Xunit;
+using TaskHouseApi.Controllers;
+using TaskHouseApi.Repositories;
+using Microsoft.AspNetCore.Mvc;
+using TaskHouseApi.Model;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace TaskHouseUnitTests 
+{ 
+    public class SkillsControllerUnitTests 
+    { 
+        SkillsController controller; 
+        ISkillRepository repo; 
+
+        public SkillsControllerUnitTests() 
+        { 
+            repo = new FakeSkillRepository();
+            controller = new SkillsController(repo);
+        }
+
+        ///Test Get all
+        [Fact]
+        public async void  SkillsController_ReturnsAllElementsInRepo() 
+        { 
+            //Arrange and act
+            IEnumerable<Skill> result = await controller.Get();
+
+            //Asserts
+            Assert.Equal(3, result.Count());
+
+        }
+
+        ///Test Get with valid Id as parameter
+        [Fact]
+        public async void SkillsController_Get_ReturnsObjectReponse_WhenGivenValidId() 
+        { 
+            //arrange
+            int skillId = 2;
+
+            //Act
+            var result = await controller.Get(skillId);
+
+            //Assert
+            var okResult = Assert.IsType<ObjectResult>(result);

```

Comment:
Maybe we should check that the returned result actually contains the expected data.